test/extensions/filters/http/ext_authz/BUILD

```diff
@@ -98,21 +99,56 @@ envoy_proto_library(
 )
 
 envoy_cc_fuzz_test(
-    name = "ext_authz_fuzz_test",
-    srcs = ["ext_authz_fuzz_test.cc"],
+    name = "ext_authz_grpc_fuzz_test",
+    srcs = ["ext_authz_grpc_fuzz_test.cc"],
     corpus = "ext_authz_corpus",
     deps = [
+        ":ext_authz_fuzz_lib",
         ":ext_authz_fuzz_proto_cc_proto",
-        "//source/common/http:context_lib",
-        "//source/common/network:address_lib",
+        "//source/common/common:assert_lib",
+        "//source/extensions/filters/common/ext_authz:ext_authz_grpc_lib",
         "//source/extensions/filters/http/ext_authz",
         "//test/extensions/filters/common/ext_authz:ext_authz_mocks",
+        "//test/extensions/filters/common/ext_authz:ext_authz_test_common",
         "//test/extensions/filters/http/common/fuzz:http_filter_fuzzer_lib",
         "//test/mocks/grpc:grpc_mocks",
-        "//test/mocks/http:http_mocks",
+        "@envoy_api//envoy/service/auth/v3:pkg_cc_proto",
+    ],
+)
+
+envoy_cc_fuzz_test(
+    name = "ext_authz_http_fuzz_test",
+    srcs = ["ext_authz_http_fuzz_test.cc"],
+    corpus = "ext_authz_corpus",
+    deps = [
+        ":ext_authz_fuzz_lib",
```

Comment:
As you put a lot of dependencies in your ext_authz_fuzz_lib already, do you need to directly include them again, like 

"//source/extensions/filters/http/ext_authz",
"//test/extensions/filters/common/ext_authz:ext_authz_mocks",
...

I would rather just include ext_authz_fuzz_lib and other dependencies which is not included in ext_authz_fuzz_lib. And let the compiler bring in those indirect dependencies to keep the BUILD target simple and clean.  Same as above.

Reply:
I found a couple I wasn't using directly in the http and grpc fuzz tests and removed them.